assets/components/LogViewer/LogActionsToolbar.vue

```diff
@@ -65,17 +65,56 @@
           {{ $t("toolbar.show", { std: "STDERR" }) }}
         </a>
       </li>
+      <li class="line"></li>
+      <!-- Container Actions (Enabled via config) -->
+      <ul v-show="enableActions">
+        <li>
+          <button
+            @click="stop"
+            :disabled="actionStates.stop || actionStates.restart"
+            v-if="container.state == 'running'"
+          >
+            <carbon:stop-filled-alt /> {{ $t("toolbar.stop") }}
+          </button>
+
+          <button
+            @click="start"
```

Comment:
I am pretty sure this might be a typescript error and it should be `@click="start()"`.